README.md

```diff
@@ -126,6 +132,13 @@ simple
 ## Examples
 Check out the [Examples](examples/) folder to see different use cases.
 
+- [Basic Setup](examples/simple)
+- [Custom Message Syntax](examples/custom-message-syntax)
+- [Identical Keys (Simple)](examples/identical-keys-simple)
+- [Ignoring Keys](examples/ignore-keys)
+- [Multiple Files Per Locale](examples/multiple-files-per-locale)
+- [Webpack Development (eslint-loader)](examples/webpack-local-dev)
```

Comment:
I'm a bit wary of the value of duplicating this list here. I think it adds unnecessarily to the maintenance burden of this document, and could easily be overlooked in the future.